contrib/openvpn/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "openvpn"
+pkgver = "2.6.8"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    "--with-openssl-engine",
+    "--with-crypto-library=openssl",
+    "--enable-x509-alt-username",
+]
+make_dir = "."
+hostmakedepends = [
+    "automake",
+    "pkgconf",
+    "libtool",
+    "python",
+    "openssl",
+    "cmocka",
+]
+makedepends = [
+    "linux-pam-devel",
```

Comment:
this list also needs linux-headers. also i think you need `depends = ` on iproute2 (it needs `ip` right?)